numpy/core/include/numpy/ndarrayobject.h

```diff
@@ -152,19 +152,16 @@ extern "C" {
                                             (k)*PyArray_STRIDES(obj)[2] + \
                                             (l)*PyArray_STRIDES(obj)[3]))
 
-/* Move to arrayobject.c once PyArray_XDECREF_ERR is removed */
```

Comment:
That comment is wrong: this function is part of the public API since it is a static function in the public headers. We could move it to a private function but only after a deprecation period ...